openhcl/virt_mshv_vtl/src/cvm_cpuid/snp.rs

```diff
@@ -436,8 +436,7 @@ impl CpuidArchInitializer for SnpCpuidInitializer {
             // Hyper-V MSRs to function. Enable it here always.
             .with_use_apic_msrs(true)
             .with_long_spin_wait_count(!0)
-            .with_use_hypercall_for_remote_flush_and_local_flush_entire(true)
-            .with_use_synthetic_cluster_ipi(true);
```

Comment:
Let's leave this for SNP, at least for now. We don't have a kernel fast path for architectural IPIs on SNP... and even if we did, the hypervisor supports an even faster path via GHCB that doesn't involve exiting to the paravisor at all.